lib/csrf.js

```diff
@@ -92,17 +92,16 @@ module.exports = function (options) {
         var shouldBypass = false;
 
         if (blocklist) {
-            blocklist.some(function (exclusion) {
-                shouldBypass = req.path.indexOf(exclusion) === 0;
-                return shouldBypass;
-            });
+            shouldBypass = (blocklist.indexOf(req.path) !== -1);
```

Comment:
this seems to change the functionality. For eg. earlier `/foo` in blocklist would return true for `/foo/` but this won't

Reply:
ah, interesting. problem we were running into is that `/` in the blocklist would put all routes into the blocklist.

Reply:
I think we can add an explicit check for `/` in config. For eg:
```
blocklist.some(function (exclusion) {
  shouldBypass = exclusion.length === 1 ? req.path === exclusion : req.path.indexOf(exclusion) === 0 :
  return shouldBypass;
})
     
```

Reply:
@grawk any thoughts?

Reply:
how about a case where the blocklist defines `/blocklist` and the route `/blocklist/extra/` is passed? 

should `/blocklist/extra/` be on the blocklist or only if `/blocklist/*` is specified?